docusaurus.config.js

```diff
@@ -50,7 +50,7 @@ module.exports = {
     announcementBar: {
       id: "camunda8",
       content:
-        '<b>📣 Camunda Cloud is now Camunda Platform 8!</b> Learn more about navigating the updated documentation <a target="_blank" rel="noopener noreferrer" href="https://docs.camunda.io/meta">here</a>.',
+        '📣 <b><a target="_blank" rel="noopener noreferrer" href="https://accounts.cloud.camunda.io/signup?uc=signup&utm_source=docs.camunda.io&utm_medium=referral&utm_content=banner">Sign-Up</a></b> for a free account to start orchestrating business processes today.',
```

Comment:
I made the URL bold because it didn't stand out quite a much, but I'm wondering if I'm just biased because the other one was bold. 🤷‍♀️